godef.go

```diff
@@ -36,20 +37,45 @@ func fail(s string, a ...interface{}) {
 }
 
 func init() {
-	// take GOPATH, set types.GoPath to it if it's not empty.
-	p := os.Getenv("GOPATH")
-	if p == "" {
-		return
+	if os.Getenv("GO15VENDOREXPERIMENT") != "" {
+		setupVendorExperiment()
```

Comment:
I'm all for it.. I just didn't want to impact the upstream too much.. but indeed, it would be better being enabled all the time.